src/parser.js

```diff
@@ -54,6 +55,14 @@ const parser = yargs
       default: false,
       type: 'boolean',
     },
+    prettier: {
+      describe: oneLine`
+          Prettier configuration options 
+          to be passed to prettier-eslint
+          using dot-notation`,
+      type: 'string',
```

Comment:
Is `type: 'string'` correct? I don't understand how the dot-notation stuff works, so I'm just confirming this wasn't an error.